collector/project.go

```diff
@@ -44,22 +46,35 @@ func (p Project) Collect(c *CollectorOpts) interface{} {
 	p.Orchestration[orchestrationName] = total
 	p.Total = total
 
+	p.LibraryCharts = make(LabelCount)
+
 	var nsUtils []float64
 	var wlUtils []float64
 	var poUtils []float64
 	var hpaUtils []float64
 
+	// Setup vars for catalogs
+	var globalRancherLibraryNames []string
+	globalCatalogs, err := c.Client.Catalog.List(&opts)
```

Comment:
Do a get for the catalog with id 'local' then validate the url is as expected. If it doesn't match then we can just skip the gathering of data.

Reply:
fixed